modules/git/repo_show.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package git
+
+import (
+	"fmt"
+)
+
+// GetFileContent returns file content for given revision.
+func (repo *Repository) GetFileContent(rev, path string) ([]byte, error) {
+	cmd := NewCommand(repo.Ctx, "show")
+	cmd.AddDynamicArguments(fmt.Sprintf("%s:%s", rev, path))
+	stdout, _, err := cmd.RunStdBytes(&RunOpts{Dir: repo.Path})
+	if err != nil {
+		return nil, err
+	}
+
+	return stdout, nil
+}
```

Comment:
why not use already exist function to read repo file? (like read file logic in viewFile)

Reply:
I looking for existing method to view files, but does not find it.
Can you please help me to find it.

Reply:
```
commit, err := gitRepo.GetBranchCommit(repo.DefaultBranch)
blob, err := commit.GetBlobByPath("README.md")
bytes, err := blob.GetBlobContent() // or blob.DataAsync() to use streaming
```

Reply:
Thank you.